<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePostWindow</class>
 <widget class="QMainWindow" name="UpdatePostWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atualizar Posto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="adressTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Endereço:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="adressLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updatePostButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar posto de saúde</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="adressTag_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Vacinas disponiveis:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="availableVaccinesLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
